solidity/contracts/BancorNetwork.sol

```diff
@@ -1,5 +1,5 @@
-pragma solidity 0.4.26;
-import "./IBancorNetwork.sol";
+// SPDX-License-Identifier: SEE LICENSE IN LICENSE
+pragma solidity >=0.6.12 <0.7.0;

```

Comment:
I'd rather stick to a specific compiler version, like before. Future 0.6.X versions may introduce undesired behaviors.

Reply:
what about interoperability?

Reply:
It won't really help since the production contracts would be still compiled with a specific 0.6.X version, which can cause issues when used with a 0.6.Y contract. You can consider switching to `npm publish`, exporting the interfaces, and then keep `pragma solidity >=0.6.12 <0.7.0` so that other projects could import them easily.

Reply:
updated the contracts to a specific version